packages/dates/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@shopify/dates",
+  "version": "0.0.0",
```

Comment:
Consult with whoever wrote the original code to see if this is in a 1.0 state.  If there are API changes outstanding, make the version <1.

Reply:
Seems like most of it hasn't been touched in about 2 months. I pinged @joshuajay to see what he thinks.